pkg/cache/cache.go

```diff
@@ -0,0 +1,15 @@
+package cache
+
+type Cache interface {
```

Comment:
Wouldn't it be more consistent with the rest of the code to rename this to `ICache`?

Reply:
Good point! I adjusted it :slightly_smiling_face:

Reply:
@matthisholleville what is the workflow here? 
Should I resolve the conversation, or are you going to do it (I would assume you click the button, since you requested the change, but I might be wrong :slightly_smiling_face: )?